specification/indices/simulate_index_template/IndicesSimulateIndexTemplateRequest.ts

```diff
@@ -42,6 +42,15 @@ export interface Request extends RequestBase {
     name: Name
   }
   query_parameters: {
+    /**
+     * Whether the index template we optionally defined in the body should only be dry-run added if new or can also replace an existing one
+     * @server_default false
+     */
+    create?: boolean
+    /** User defined reason for dry-run creating the new template for simulation purposes
+     * @server_default false
+     */
+    cause?: string
```

Comment:
Those parameters are read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestSimulateIndexTemplateAction.java#L57-L58 since at least [7.8.0](https://github.com/elastic/elasticsearch/pull/55936).